cmd/olympus/Dockerfile

```diff
@@ -8,9 +8,8 @@ FROM gobuffalo/buffalo:v0.11.0 as builder
 RUN mkdir -p $GOPATH/src/github.com/gomods/athens
 WORKDIR $GOPATH/src/github.com/gomods/athens
 ADD . .
-WORKDIR $GOPATH/src/github.com/gomods/athens/cmd/olympus
 
-RUN buffalo build -s -o /bin/app
+RUN cd cmd/olympus && npm install && buffalo build -s -o /bin/app
```

Comment:
@marstr I think you can do a `yarn install` above this line, similar to what the proxy does in https://github.com/gomods/athens/blob/master/cmd/proxy/Dockerfile#L11

What do you think?

Reply:
Fixed in 484fd0e.